public/discord.css

```diff
@@ -1098,6 +1098,42 @@
   .discord-view strong {
     font-weight: 600
   }
+
+  .discord-view h1 {
```

Comment:
some comment (start/end region) to make it clear these parts were added manually in case we ever need a new discord.css